oslabs-beta/Kubiquity
# Add cpu component

WHAT DOES THIS PR DO:
1. Connect the FE to the BE for the CPU metrics. 
2. Create new FE component for the CPU metrics. 

HOW TO TEST THIS PR:
1. Open Docker Desktop
2. In your terminal, run `minikube start`
3. `gh pr checkout 38 && npm install && npm run dev`
4. Verify that the new component loads with the correct data 

WHAT ELSE TO NOTE: 
1. Sometimes it takes my app a few refreshes to get the metrics data. This might be the result of only ever using K8s when I'm testing our app.
